forge-gui/res/cardsfolder/m/monkey_cage.txt

```diff
@@ -5,4 +5,5 @@ T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creatu
 SVar:TrigSac:DB$ Sacrifice | Defined$ Self | SubAbility$ DBToken | RememberSacrificed$ True
 SVar:DBToken:DB$ Token | ConditionDefined$ Remembered | ConditionPresent$ Card.Self | TokenAmount$ X | TokenScript$ g_2_2_monkey | TokenOwner$ You
 SVar:X:TriggeredCard$CardManaCost
+DeckHas:Ability$Sacrifice|Token & Type$Monkey
```

Comment:
also Color$Green ?? I'm unsure

Reply:
You mean red, I was thinking about that as well